medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Medicinskiy Academicheskiy Joural</title>
```

Comment:
Yes, this is "Medicinskiy Akademicheskiy Zhurnal". This transliteration is more correct.